core/src/test/scala/neotypes/PathSessionSpec.scala

```diff
@@ -21,11 +21,9 @@ class PathSessionSpec extends BaseIntegrationSpec[Future] {
     }
   }
 
-  it should "assign path to case class field" ignore execute { s =>
-    for {
-      data <- "match path=(_:Person)-[*]->() return { path: path }".query[Data].single(s)
-    } yield {
-      assert(data.path.nodes.size == 2)
+  it should "assign path to case class field" in execute { s =>
+    "match path=(_:Person)-[*]->() return { path: path }".query[Data].list(s).map{
+      res => assert(res.head.path.nodes.size == 2)
```

Comment:
We have been having this convention over all the project.

```scala
// If just one named param.
foo.bar { param =>
   body
}

// If using pattern matching.
foo.bar {
  case param =>
     body
}
```

So, in this case, please move the parameter to the end of the previous line and please add a space between `map` and `{`.

Reply:
> We have been having this convention over all the project.
> 
> ```scala
> // If just one named param.
> foo.bar { param =>
>    body
> }
> 
> // If using pattern matching.
> foo.bar {
>   case param =>
>      body
> }
> ```
> 
> So, in this case, please move the parameter to the end of the previous line and please add a space between `map` and `{`.

@BalmungSan made the formatting fixes and also added more assertions to make it clear what the result set returned is.